jekyll/_cci2/configuration-reference.md

```diff
@@ -341,12 +341,19 @@ jobs:
 ##### Available `machine` images
 CircleCI supports multiple machine images that can be specified in the `image` field:
 
+* `ubuntu-2004:202101-01` - Ubuntu 20.04, Docker v20.10.2, Docker Compose v1.28.2,
 * `ubuntu-2004:202010-01` - Ubuntu 20.04, Docker v19.03.13, Docker Compose v1.27.4, `ubuntu-2004:202008-01` is an alias
+
+* `ubuntu-1604:202101-01` - Ubuntu 16.04, Docker v20.10.2, Docker Compose v1.28.2, 2nd to last release
```

Comment:
Should this be Docker v19.03.14?

Reply:
Correct. I doubled checked [here](https://discuss.circleci.com/t/linux-machine-executor-images-2021-january-q1-update/38976) and in a test build.